pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/ValidatorTest.java

```diff
@@ -882,4 +895,34 @@ public void bigIntegerFailure() throws Exception {
                 .logContains(Messages.ModelParser_BigIntegerValue())
                 .go();
     }
+
+    @Test
+    public void enableOptionalValidator() throws Exception {
+        String script = pipelineSourceFromResources("simplePipeline");
+
+        assertNotNull(Converter.scriptToPipelineDef(script));
+
+        thrown.expect(MultipleCompilationErrorsException.class);
+        thrown.expectMessage(Matchers.containsString("Echo is banned"));
+        Converter.scriptToPipelineDef(script, Collections.<Class<? extends DeclarativeValidatorContributor>>singletonList(RejectEchoStep.class));
```

Comment:
So no user facing way of enabling this? Only code?

Reply:
Correct.